lib/models/user/user_info.g.dart

```diff
@@ -20,11 +20,10 @@ class UserAdapter extends TypeAdapter<User> {
       authToken: fields[1] as String,
       email: fields[3] as String,
       name: fields[2] as String,
-      location: fields[6] as Location,
```

Comment:
removing a parameter doesn't seem like the way to fix the error message. Moreover this field defines a user and will be useful in the app